.ci/acceptance-tests/inspec-post-merge.sh

```diff
@@ -56,17 +56,17 @@ export INSPEC_DIR=${PWD}
 trap cleanup EXIT
 
 set +e
-gsutil ls gs://magic-modules-inspec-bucket/$PR_ID/approved
+gsutil ls gs://magic-modules-inspec-bucket/$PR_ID/inspec-cassettes/approved
 if [ $? -eq 0 ]; then
 	# We have already recorded new cassettes during the inspec-post-merge step
-	gsutil cp gs://magic-modules-inspec-bucket/$PR_ID/inspec-cassettes/approved/* gs://magic-modules-inspec-bucket/master/inspec-cassettes
+	gsutil -m cp gs://magic-modules-inspec-bucket/$PR_ID/inspec-cassettes/approved/* gs://magic-modules-inspec-bucket/master/inspec-cassettes
 else
 	# We need to record new cassettes for this PR
 	bundle exec rake test:integration
-	gsutil cp inspec-cassettes/* gs://magic-modules-inspec-bucket/master/inspec-cassettes/
+	gsutil -m cp inspec-cassettes/* gs://magic-modules-inspec-bucket/master/inspec-cassettes/
 fi
 set -e
 
 # Clean up cassettes for merged PR
-gsutil rm gs://magic-modules-inspec-bucket/$PR_ID/inspec-cassettes/*
+gsutil -m rm -r gs://magic-modules-inspec-bucket/$PR_ID/inspec-cassettes/*
```

Comment:
The -r flag scares me a little bit. Why wasn't it necessary before?

Reply:
Have a subfolder that gets deleted too. Everything in the folder of /$PR_ID should be deletable